### PR TITLE
Allows amount and phone number to be obtained from API call

### PR DESCRIPTION
**Pull request by doctortickle**
The amount of the transaction seems like very useful information to be able to get from a list of transactions. I have added this functionality.

### CODE REVIEW

File: venmo_api/models/json_schema.py [hunk 1]
```diff
@@ -62,6 +62,8 @@ def get_target(self):
     def get_status(self):
         return self.payment[payment_json_format['status']]
 
+    def get_amount(self):
+        return self.payment[payment_json_format['amount']]
```
Comment (mmohades) [inline venmo_api/models/json_schema.py:66]: Use this instead: `return self.payment.get(payment_json_format['amount'])`

**Comment by mmohades** [comment]
Thanks for opening a pull request. I totally forgot about the amount of the transaction. Can you please check my comment, and add `amount` to the string method ` __str__` as well?

**Comment by doctortickle** [comment]
Updated to meet your requirements!

**Comment by doctortickle** [comment]
I added phone number to user and profile in the same way.

**Pull request merged**